priority_alerts/views.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import (
+    permissions, generics
+)
+
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework_gis import filters as gis_filters
+from priority_alerts import (
+    serializers,
+    models,
+    filters as alerts_filters
+)
+
+
+class AuthModelMixIn:
```

Comment:
MIssing docstrings in file.

Reply:
Inserted temporary docstrings tha will be adjusted in the next branch.